extensions.yml

```diff
@@ -68,3 +68,6 @@ flask-mongoengine:
 flask-avatars:
   repo: helloflask/flask-avatars
   init_string: 'from flask_avatars import Avatars; Avatars(app)'
+flask-smorest:
+  repo: marshmallow-code/flask-smorest
+  init_string: 'from flask_smorest import Api; Api(app)'
```

Comment:
As the [CI log](https://github.com/greyli/flask-extension-status/actions/runs/7015716647/job/19090785361?pr=7) indicates, please add the `title` param.

Reply:
[344abf2](https://github.com/greyli/flask-extension-status/pull/7/commits/344abf2c333df10a124c5b461fecf17ce2468bb2)

Reply:
sorry i've missed that (also squashed all PR commits)

Reply:
also changed string quoting, it should produce correct workflow config